### PR TITLE
Create 109-ipfs-meetup-sep2020.md

### PR DESCRIPTION
**Pull request by autonome**

### CODE REVIEW

File: content/post/109-ipfs-meetup-sep2020.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+The sole brave lightning demoer was Jessica Schilling, who has been working on a series of case studies from projects who are building and shipping products and services on IPFS. Come for the dweb, stay for the rhymes.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/5qe6H77PflU" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## 🎉 Thanks again!
+
+Thanks to everyone who joined, all our speakers and everyone who's reading this or watching the IPFS channel on YouTube!
+
+See you in October... and if you cannot wait until then, [subscribe to the IPFS weekly newsletter](http://eepurl.com/gL2Pi5) to get releases, project announcements and job opportunities in your inbox 💌.
```
Comment (jacobheun) [inline content/post/109-ipfs-meetup-sep2020.md:43]:
```suggestion
Have a project you want to share with the community? [Send us an email](mailto:ipfs-community@protocol.ai) to sign up for a talk or lightning demo at an upcoming meetup!

See you in October... and if you cannot wait until then, [subscribe to the IPFS weekly newsletter](http://eepurl.com/gL2Pi5) to get releases, project announcements and job opportunities in your inbox 💌.
```

**Review by jacobheun — APPROVED** [review]
LGTM, just one suggestion for a call to action to signup for next month.

**Comment by jennwrites** [comment]
I love it, ship it with or without my review ;)

File: content/post/109-ipfs-meetup-sep2020.md (hunk 1 above)
Comment (autonome) [reply content/post/109-ipfs-meetup-sep2020.md]: good call, added, thanks!

**Pull request merged**